src/artGenerator/package.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "artgenerator",
+  "version": "1.0.0",
+  "description": "Generate art images.",
+  "main": "index.js",
```

細かいですが、ドキュメンテーションのため
```suggestion
  "main": "main.ts",
```